### ISSUE TITLE
Accessibility Frame of disabled Button too small

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11991]
When a button is disabled, it will have the accessibility box of its child, which is incorrect. It should have the same box it had when it was enabled.

Problem analysis:
The accessibility box of an enabled button is determined by the `RenderSemanticsGestureHandler`. If the button is disabled, the `RenderSemanticsGestureHandler` claims that it has no semantics to add and it's left to the child to determine the accessibility box.

**Comment by goderbauer** [comment]
Probably requires https://github.com/flutter/flutter/issues/12589 first.
